Tests/DataPipeline/APITest.swift

```diff
@@ -31,6 +36,47 @@ class DataPipelineAPITest: UnitTest {
             config.autoAddCustomerIODestination = false
         }
 
+        // Identify
+        CustomerIO.shared.identify(userId: "")
+        mock.identify(userId: "", traits: nil)
```

Comment:
We have to pass `nil` explicitly for mock, I don't think that's a problem as actual public methods are callable due to default values. Let me know if someone thinks we need to add methods explicitly for this.